simulation/eor_bubble_sim.pro

```diff
@@ -0,0 +1,84 @@
+FUNCTION eor_bubble_sim, obs, jones, select_radius=select_radius, bubble_fname=bubble_fname, beam_threshold=beam_threshold, allow_sidelobe_sources=allow_sidelobe_sources, dat=dat 
+
+;Opening an HDF5 file and extract relevant data
+if keyword_set(bubble_fname) THEN hdf5_fname = bubble_fname ELSE hdf5_fname = '/users/alanman/data/alanman/BubbleCube/TiledHpxCubes/mwa_gaussian.hdf5'
```

Comment:
Please remove hardcoding of your path, and add an error message if the bubble_fname fails a file_test.